src/client/dash.ts

```diff
@@ -50,7 +50,66 @@ function eventTemplate(event: PullRequestEvent) {
     <div class="pr-event__title">${event.text}</div>`;
 }
 
-function prBaseTemplate(pr: PullRequest, status: PullRequestStatus) {
+function outgoingStatusToDisplay(pr: OutgoingPullRequest) {
+  if (pr.status === PullRequestStatus.WaitingReview) {
+    const reviewAuthors = pr.reviews.map((review) => review.author);
+    const reviewers = pr.reviewRequests.concat(reviewAuthors);
+    return {actionable: false, text: `Waiting on ${reviewers.join(', ')}`};
+  }
+  return statusToDisplay(pr);
+}
+
+function statusToDisplay(pr: PullRequest):StatusDisplay {
+  let text = '';
+  let actionable = false;
+  switch (pr.status) {
+    case PullRequestStatus.Unknown:
+      text = '';
+      break;
+    case PullRequestStatus.NoActionRequired:
+      text = 'No action required';
+      break;
+    case PullRequestStatus.NewActivity:
+      text = 'New activity';
+      break;
+    case PullRequestStatus.StatusChecksPending:
+      text = 'Status checks pending';
+      break;
+    case PullRequestStatus.WaitingReview:
+      text = 'Waiting on reviewers';
+      break;
+    case PullRequestStatus.PendingChanges:
+      text = 'Waiting on you';
```

Comment:
I think I'd just do e.g. `return {text: 'Waiting on you', actionable: false};` in all of these cases, seems a bit simpler, and also makes it harder to accidentally forget a `break`.

Reply:
Good point. I forgot to change it to that cause I had it inlined before instead of a function.

Done.